pycon.md

```diff
@@ -12,74 +12,86 @@ revealOptions:
 
 ## Ending Python2 compatibility in a user friendly manner
 
-M. Bussonnier & M. Pacer
+M. Bussonnier & M Pacer
 
 Slides available at https://short.url
 
 ----
 
 # About us
 
-We have been Both Working on the IPython and  Jupyter Project for respectively
-5 and 1 year. 
+We have been working on the IPython and Jupyter Projects for
+5 years and ~1 year, respectively . 
 
 github:@Carreau/twitter:@Mbussonn
 
-
-github:@mpacer/twitter:@
+github:@mpacer/twitter:@mdpacer
 
 ----
 
-## What this talks is **not** about
+## What this talk is **not** about
 
-- Is Python 2 or 3 the right choice 
-- Should I migrate to Python3-only
+- Is Python 2 or 3 the right choice? 
+- Should I migrate to Python3-only?
 
 --
 
-## What this talk is about
+## What this talk **is** about
+
+We migrated IPython to Python 3 only. 
+
+We care about *all* of our users, Python 2 and 3 alike.
 
-We decided to migrate IPython to Python 3 only
+We want to make the transition the least frustrating for users and dev.
 
-Ho to make the transition the least frustrating for users and dev.
+We'll be describing how we did this.
 
 --
 
-## 2 vs 3 is an **example**
+## Python 2 vs 3 is an **example**
 
-This talks does apply to stopping support for 2.6, or 3.3
+The lessons we've learned are not specific a python2 to python3 transition. 
+
+Our talk applies to stopping support for any version (e.g., 2.6 or 3.3).
 
 ----
 
 # Python 3 Statement
 
 [www.python3statement.org](http://www.python3statement.org/)
 
-List of who is stopping support when. 
+List of who is stopping Python 3 support when. 
 Resources on how to stop support with minimal frustration for users
 
 --
 
 ![p3s-list](p3s-list.png)
 
+Note: We should have text notes here to simplify reading this off.
+
 --
 
 ![p3s-list](p3s-timeline.png)
 
+Note: We should have text notes here to simplify reading this off.
+
 ----
 
 # Scratch your own itch
 
-We want to release IPython 6, the code base will be Python 3 only. 
-IPython 5 will still be stable (LTS). So we'd like Python 2 user to
+We released IPython 6, the code base is Python 3 only.
+
+IPython 5 will still be stable (LTS). So if a Python 2 user runs
+
+    $ pip install ipython -U
 
-    $ pip install ipython
+it should install the latest version of IPython 5, not IPython 6!
 
 --
 
 ## Core of the problem
 
-```python
+```bash
```

Comment:
I'm applying the modification manualy here are some reason why I don't apply some. 

bash highlighting is wrong. Python one looks correct.